src/sage/graphs/graph.py

```diff
@@ -9612,6 +9612,8 @@ def bipartite_double(self, extended=False):
     from sage.graphs.graph_coloring import fractional_chromatic_number
     from sage.graphs.graph_coloring import fractional_chromatic_index
     from sage.graphs.hyperbolicity import hyperbolicity
+    from sage.graphs.matching import (has_perfect_matching, matching,
```

Comment:
please do 1 line per import, as for other methods around. Add them in the doc as well. `is_bicritical` is missing.

Reply:
Shall I add include these methods under `Matching` or under `Leftovers` in the doc of `graph.py`?

Reply:
Matching is certainly appropriate.